server/controllers/userController.js

```diff
@@ -38,8 +38,6 @@ exports.updateUserInfo = async function (req, res) {
       },
       {
         $set: {
-          name: req.body.name,
-          avatarUrl: req.body.avatarUrl,
           timeZone: req.body.timeZone,
```

Comment:
could you please add/update other info's to the user database as well. The access_token, refresh_token, and expire_at are returned to the login/signup page with the user profile.

Reply:
Yes I can but don't we have to encrypt it before using JWT. I could be wrong let me know.

Reply:
Seems like JWT web tokens are encouraged to encrypted, but not sure if the google refrsh_code needs to be encrypted as well. @bonnieli